clr-installer-ci/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM clearlinux:latest
+MAINTAINER mark.d.horn@intel.com
+
+# Configure Go
+ENV GOPATH /go
+ENV PATH "/go/bin:${PATH}"
+
+# Configure trust
+RUN clrtrust generate
+# Update and add bundles
+RUN swupd update
+RUN swupd bundle-add sysadmin-basic storage-utils network-basic go-basic-dev git telemetrics
+# Install the Go Linters
+RUN go get -u gopkg.in/alecthomas/gometalinter.v2
+RUN gometalinter.v2 --install
```

Comment:
I strongly encourage you to join these together into a single `RUN`:
```dockerfile
# Configure trust
RUN clrtrust generate && \
    # Update and add bundles
    swupd update && \
    swupd bundle-add sysadmin-basic storage-utils network-basic go-basic-dev git telemetrics && \
    # Install the Go Linters
    go get -u gopkg.in/alecthomas/gometalinter.v2 && \
    gometalinter.v2 --install
```

The reason is that, as is, every single `RUN` step builds a _separate layer_ during the `docker build` step. This significantly increases the size and time needed to do the build.

This does come at the expense of it being more difficult to tell which step caused the build failure if the error output is generic.

Reply:
@kevincwells Is that legal syntax to have the comments in between the commands like that?
I'll go do a quick test for sizing.

Reply:
@mdhorn Yup, should work

Also, to be clear -- the _final image size_ is not increased, just the size needed for the overall build. As this is all going to take place on DockerHub, this is a suggestion and not a requirement. The only real benefit you'll see is faster builds. (Though you'd see the space savings on your local machine if doing test builds.)

Reply:
Actually, why you're at it, you can combine the `ENV`'s too. The syntax to set multiple in a single line is:

```dockerfile
ENV GOPATH=/go PATH="/go/bin:${PATH}"
```